wpilibj/src/test/java/edu/wpi/first/wpilibj/smartdashboard/SendableChooserTest.java

```diff
@@ -85,6 +85,25 @@ void testChangeListener() {
     }
   }
 
+  @ValueSource(ints = {0, 1, 2, 3})
+  @ParameterizedTest
+  void testSendableChooserSim(int toSelect) {
+    try (var chooser = new SendableChooser<Integer>();
+        var chooserSim =
+            new SendableChooserSim(m_inst, "/SmartDashboard/SendableChooserSim" + toSelect + "/")) {
+      for (int i = 1; i <= 3; i++) {
+        chooser.addOption(String.valueOf(i), i);
+      }
+      chooser.setDefaultOption(String.valueOf(0), 0);
+
+      SmartDashboard.putData("SendableChooserSim" + toSelect, chooser);
+      SmartDashboard.updateValues();
+      chooserSim.setSelected(String.valueOf(toSelect));
+      SmartDashboard.updateValues();
+      assertEquals(toSelect, chooser.getSelected());
+    }
+  }
+
```

Comment:
Should C++ have an equivalent test?

Reply:
No. I think this test was leftover from a previous implementation. The current tests have been modified now to just use SendableChooserSim.